vernac/vernacentries.ml

```diff
@@ -1656,6 +1651,15 @@ let () =
       optread  = CDebug.get_flags;
       optwrite = CDebug.set_flags }
 
+let pwd_key = ["Pwd"]
+
+let () =
+  declare_string_option
+    { optdepr  = false;
+      optkey   = pwd_key;
+      optread  = Sys.getcwd;
+      optwrite = set_pwd }
```

Comment:
needs doc
Also not sure if it makes sense to have this be an option, do we really want the possibility of Require doing chdir?

Reply:
> do we really want the possibility of Require doing chdir?

I would say no.

But there is still the question of synchronization with backtracking. E.g. if I do:
```coq
Goal True.
Pwd.
Cd "/".
Pwd.
```
Then going to the end then backtracking on the first `Pwd` gives `/`.

> needs doc

Yes, this should be part of #14720.